Apps/WebClient/src/ClientApp/src/components/monthYearPicker.vue

```diff
@@ -90,25 +90,16 @@ export default class MonthYearPickerComponent extends Vue {
         return this.selectedDate.format("MMMM yyyy");
     }
 
-    private getDisplayYearCss(year: number) {
-        return `item col-12 ${
-            this.currentMonth.year() === year ? "selected" : ""
-        }`;
+    private isCurrentYear(year: number): boolean {
```

Comment:
private get ...

Reply:
just saw the parameter so idk if this works with a getter

Reply:
same as the previous one, it needs the year parameter